config/locales/en.yml

```diff
@@ -248,16 +248,19 @@ en:
         calculate_range: Calculate the end of the batch by
         select_range: Select range
       label:
-        week_condition: Weekly rhythm
-        week_of_month: Week of the month
-        period: Interval
-        date_range_condition: End range
+        week_condition: Calculate the weeks by
+        week_of_month: Select Week of the month
+        period: Select weekly interval
+        date_range_condition: Calculate the end of the batch by
     form_side:
       header:
-        preview: Schedule Preview
-        publish_time: Publish Time
-        audio_segments: Audio Segments
-        number_of_drafts: "Number of Draft Episodes: %{count}"
+        preview: Preview Draft Schedule
+        publish_time: What time of day do you plan to publish the episodes?
+        audio_segments: How many audio segments?
+        segment_count: Number of segments
+        number_of_drafts: "Drafts planned: "
```

Comment:
changed this so it no longer needs to pass in a `count` variable. the count is displayed in the view in the innerHTML of a new span in the h3 that uses this label.